### ISSUE TITLE
Docker-compose does not work well

### ISSUE THREAD

**Issue by shlomi-nx** [nutanix/papiea-js#24]
Here is the results of running this in my aws instance:

```bash
~/work/papiea-js (master) > docker-compose up
Starting papiea-js_server_1_eb7d95c6ee57 ... done
Attaching to papiea-js_server_1_eb7d95c6ee57
server_1_eb7d95c6ee57 |
server_1_eb7d95c6ee57 | > papiea-core@1.0.0 build-clj /code
server_1_eb7d95c6ee57 | > cd papiea-lib-clj && lein cljsbuild once
server_1_eb7d95c6ee57 |
server_1_eb7d95c6ee57 | Compiling ClojureScript...
server_1_eb7d95c6ee57 |
server_1_eb7d95c6ee57 | > papiea-core@1.0.0 build /code
server_1_eb7d95c6ee57 | > tsc
server_1_eb7d95c6ee57 |
server_1_eb7d95c6ee57 | sh: 1: tsc: not found
server_1_eb7d95c6ee57 | npm ERR! file sh
server_1_eb7d95c6ee57 | npm ERR! code ELIFECYCLE
server_1_eb7d95c6ee57 | npm ERR! errno ENOENT
server_1_eb7d95c6ee57 | npm ERR! syscall spawn
server_1_eb7d95c6ee57 | npm ERR! papiea-core@1.0.0 build: `tsc`
server_1_eb7d95c6ee57 | npm ERR! spawn ENOENT
server_1_eb7d95c6ee57 | npm ERR!
server_1_eb7d95c6ee57 | npm ERR! Failed at the papiea-core@1.0.0 build script.
server_1_eb7d95c6ee57 | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
server_1_eb7d95c6ee57 | npm WARN Local package.json exists, but node_modules missing, did you mean to install?
server_1_eb7d95c6ee57 |
server_1_eb7d95c6ee57 | npm ERR! A complete log of this run can be found in:
server_1_eb7d95c6ee57 | npm ERR!     /root/.npm/_logs/2018-12-11T21_41_21_687Z-debug.log
server_1_eb7d95c6ee57 | internal/modules/cjs/loader.js:583
server_1_eb7d95c6ee57 |     throw err;
server_1_eb7d95c6ee57 |     ^
server_1_eb7d95c6ee57 |
server_1_eb7d95c6ee57 | Error: Cannot find module '/code/lib/main.js'
server_1_eb7d95c6ee57 |     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
server_1_eb7d95c6ee57 |     at Function.Module._load (internal/modules/cjs/loader.js:507:25)
server_1_eb7d95c6ee57 |     at Function.Module.runMain (internal/modules/cjs/loader.js:742:12)
server_1_eb7d95c6ee57 |     at startup (internal/bootstrap/node.js:282:19)
server_1_eb7d95c6ee57 |     at bootstrapNodeJSCore (internal/bootstrap/node.js:743:3)
papiea-js_server_1_eb7d95c6ee57 exited with code 1
```

**Comment by shlomi-nx** [comment]
Did you check this? Is it only misbehaving on my machine?

**Comment by adolgarev** [comment]
this was already fixed, please try commit 0f42c1f17f9350d910c72569709ed6234219e2fb

**Comment by shlomi-nx** [comment]
Oh I see.. We should have commits or pull requests that fixes issues to be linked, otherwise it is quite difficult to follow. I will test and close if fixed.
